framework/src/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -1680,6 +1680,12 @@ public final String getOption(String name) {
     // TODO I would like to rename getLintOption to hasLintOption
     @Override
     public final boolean hasOption(String name) {
+
+        // Always ignore errors from raw type arguments.
+        // See https://github.com/typetools/checker-framework/issues/635
+        if (name.equals("ignoreRawTypeArguments")) {
```

Comment:
Yes, but by doing it like this those call sites are confusing and there is no way to provide the alternative behavior.